src/libp2p_bitswap/request_manager.rs

```diff
@@ -162,6 +163,7 @@ impl BitswapRequestManager {
         store: Arc<impl BitswapStoreReadWrite>,
         cid: Cid,
         deadline: Instant,
+        validate_peer: Option<Arc<dyn Fn(PeerId) -> bool + Send + Sync>>,
```

Comment:
For readability, I'd make it a named type, e.g., `PeerValidateCallback`. What do you think?

Reply:
Fixed